### ISSUE TITLE
7$: E024 mentions 'let' even when using 'var' or 'const'

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#234]
```js
const
if (true) {}
let
if (true) {}
var
if (true) {}
```
Observed:
```
hello.js:1:1: error: let with no bindings [E024]
hello.js:5:1: error: let with no bindings [E024]
hello.js:3:1: warning: use of undeclared variable: let [E057]
```
Expected:
```
hello.js:1:1: error: const with no bindings [E024]
hello.js:5:1: error: var with no bindings [E024]
hello.js:3:1: warning: use of undeclared variable: let [E057]
```

**Comment by LeeWannacott** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.
  
``` cpp
QLJS_ERROR_TYPE(                                                             \
      error_let_with_no_bindings, "E024", { source_code_span where; },         \
-      .error(QLJS_TRANSLATABLE("let with no bindings"), where))               \
+     .error(QLJS_TRANSLATABLE("{0} with no bindings"), where))```           \

I changed this one line and it does the `Expected` output. 
I don't expect payment if it was as simple as changing one line.

**Comment by LeeWannacott** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).
